livekit-agents/livekit/agents/multimodal/multimodal_agent.py

```diff
@@ -248,9 +267,11 @@ def _input_speech_transcription_completed(
             user_msg = ChatMessage.create(
                 text=ev.transcript, role="user", id=ev.item_id
             )
-            self._session._update_conversation_item_content(
-                ev.item_id, user_msg.content
-            )
+
+            if self._model.capabilities.supports_chat_ctx_manipulation:
+                self._session._update_conversation_item_content(
```

Comment:
`_update_conversation_item_content` isn't making any network calls, why does it need to be guarded with `supports_chat_ctx_manipulation`

Reply:
I thought it's unrelated to gemini, because we are not maintaining transcript or chatctx, maybe we should move `_RemoteConversationItems` class to baseclass?

wdyt @theomonnom